contributingGuides/STYLE.md

```diff
@@ -341,105 +886,52 @@ So, if a new language feature isn't something we have agreed to support it's off
 Here are a couple of things we would ask that you *avoid* to help maintain consistency in our codebase:
 
 - **Async/Await** - Use the native `Promise` instead
-- **Optional Chaining** - Use `lodashGet()` to fetch a nested value instead
-- **Null Coalescing Operator** - Use `lodashGet()` or `||` to set a default value for a possibly `undefined` or `null` variable
+- **Optional Chaining** - Yes, don't use `lodashGet()`
+- **Null Coalescing Operator** - Yes, don't use `lodashGet()` or `||` to set a default value for a possibly `undefined` or `null` variable
```

Comment:
```suggestion
```

Reply:
This doesn't make sense with this header: "Here are a couple of things we would ask that you *avoid* to help maintain consistency in our codebase:"